### PR TITLE
docs - awscc_sso_permission_set

### PR DESCRIPTION
**Pull request by novekm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_sso_permission_set/awscc_sso_permission_set.tf [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+data "aws_ssoadmin_instances" "example" {} // fetch IAM Identity Center instance arn
+
+// create new permission set
+resource "awscc_sso_permission_set" "example" {
+  instance_arn = data.aws_ssoadmin_instances.example.arns[0] // reference existing IAM IDC instance by arn
+  name         = "ExamplePermissionSet"                      // add desired name for permission set
+  description  = "An example Permission Set"                 // add desired description for permission set
+  // add multiple managed policies
+  managed_policies = [
+    "arn:aws:iam::aws:policy/AdministratorAccess",
```
Comment (wellsiau-aws) [inline examples/resources/awscc_sso_permission_set/awscc_sso_permission_set.tf:23]: conflict with the view only policy, I suggest to remove it

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
thanks for the contribution, I added a few suggestion for changes

File: examples/resources/awscc_sso_permission_set/awscc_sso_permission_set.tf (hunk 1 above)
Comment (novekm) [reply examples/resources/awscc_sso_permission_set/awscc_sso_permission_set.tf]: Updated and pushed changes 👍🏾

File: examples/resources/awscc_sso_permission_set/awscc_sso_permission_set.tf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+    awscc = {
+      source = "hashicorp/awscc"
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (novekm) [reply examples/resources/awscc_sso_permission_set/awscc_sso_permission_set.tf]: Updated and pushed changes 👍🏾

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thanks!

**Pull request merged**